docs/content/tutorials/nginx-ingress-osm.md

```diff
@@ -0,0 +1,456 @@
+---
+title: NGINX Ingress Controller and Open Service Mesh
+description: |
+  Use NGINX Ingress Controller with Open Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1181"
+---
+
+## This document outlines how to integrate F5 NGINX Ingress Controller with Open Service Mesh (OSM)
+
+Open Service Mesh will work with both versions of [F5 NGINX Ingress controller](https://github.com/nginxinc/kubernetes-ingress): the free as well as the NGINX Plus versions.
+
+Below is a link to the official F5 NGINX Ingress Controller documentation.
+[F5 NGINX Ingress controller](https://docs.nginx.com/nginx-ingress-controller/)
+
+# Integrating NGINX Ingress Controller with Open Service Mesh
+
+There are two ways to integrate the NGINX Ingress Controller with Open Service Mesh (OSM):
+
+1. Injecting an envoy sidecar directly with NGINX Ingress Controller.
+2. Using the Open Service Mesh `ingressBackend` "proxy" feature.
+
+
+# NGINX Ingress controller and OSM with sidecar injected
+
+Install OSM in the cluster
+
+```bash
+osm install --mesh-name osm-nginx --osm-namespace osm-system
+```
+
+### Mark F5 NGINX Ingress controller namespace for sidecar injection
+
+*NOTE:* Depending on how you install NGINX Ingress controller, you might need to create the `namespace`. For example, if you are using manifests to install NGINX Ingress controller, you can complete all of the steps on our documentation page, *EXCEPT*, actually deploying NGINX Ingress controller. This is because, when using the sidecar approach, OSM needs to "manage" the namespace so it knows what `namespaces` it needs to inject sidecars into.
+
+Next thing we need to do is install OSM into the `NGINX Ingress controller` namespace so that the `envoy` sidecar will be injected into NGINX Ingress controller.
+First, create the `nginx-ingress` namespace:
+
+```bash
+kubectl create ns nginx-ingress
+```
+Then "mark" the `nginx-ingress` namespace for OSM to deploy a sidecar.
+
+```bash
+osm namespace add nginx-ingress --mesh-name osm-nginx
+```
+
+The above command will use the mark the `nginx-ingress` namespace, where OSM will be installed (sidecar)
+
+# Install F5 NGINX Ingress controller. Here the links to the install guides:
```

Comment:
Install F5 NGINX Ingress controller. Here the links to the install guides:
That ^ reads very awkwardly as a two-sentence headline.  Not to mention the second sentence is incomplete.

This could be simplified to:
# Install F5 NGINX Ingress controller. 

Links to the complete install guides: